supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -319,6 +352,63 @@ Resources:
                   - states:StartExecution
                 Resource: !Ref SupporterProductData
 
+  SupporterProductDataDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub dead-letters-supporter-product-data-${Stage}
+      Tags:
+        - Key: Stage
+          Value: !Ref Stage
+
+  SupporterProductDataQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub supporter-product-data-${Stage}
+      RedrivePolicy:
+        deadLetterTargetArn:
+          Fn::GetAtt:
+            - "SupporterProductDataDeadLetterQueue"
+            - "Arn"
+        maxReceiveCount: 10
+      VisibilityTimeout: 3000
+      Tags:
+        - Key: Stage
+          Value: !Ref Stage
+
+  SQSTrigger:
+    Type: AWS::Lambda::EventSourceMapping
+    Properties:
+      BatchSize: 1
+      Enabled: true
+      EventSourceArn: !GetAtt SupporterProductDataQueue.Arn
+      FunctionName: !Ref ProcessSupporterRatePlanItemLambda
+    DependsOn:
+      - SupporterProductDataQueue
+      - ProcessSupporterRatePlanItemLambda
+
+  UnprocessedSupporterProductDataRecordAlarm:
+    Type: 'AWS::CloudWatch::Alarm'
+    Condition: CreateProdResources
+    Properties:
+      AlarmName: !Sub 'Urgent 9-5 - There was a failure processing a supporter record in the ProcessSupporterRatePlanItemLambda lambda'
+      AlarmDescription: >
+        Check the dead-letters-direct-mail-PROD SQS queue for details of the record which failed and the ProcessSupporterRatePlanItemLambda
+        Cloudwatch logs for details of what went wrong
```

Comment:
would be ideal to have a "see this doc for more about what it means and how to fix it"